reactapp/src/pages/SensorData.jsx

```diff
@@ -90,6 +100,70 @@ export default function SensorData() {
 
   const [wifiData, setWifiData] = useRecoilState(wifiState);
 
+  // eslint-disable-next-line react/no-unstable-nested-components
+  function TextReader() {
+    return (
+      <div>
+        <p className="field_name" mb={10}>
+          Include or exclude specific package to study *
+        </p>
+        <Grid marginTop={2}>
+          <RadioGroup
+            aria-labelledby="package_specification"
+            name="package_specification"
+            value={applicationSensor.package_specification || "2"}
+            row
+          >
+            <FormControlLabel
+              value="0"
+              control={<Radio />}
+              label="Inclusive packages"
+              onClick={(_, checked) => {
+                updateApplicationSensorData("package_specification", "0");
+              }}
+            />
+            <FormControlLabel
+              value="1"
+              control={<Radio />}
+              label="Exclusive packages"
+              onClick={(_, checked) => {
+                updateApplicationSensorData("package_specification", "1");
+              }}
+            />
+            <FormControlLabel
+              value="2"
+              control={<Radio />}
+              label="Default track all packages"
+              onClick={(_, checked) => {
+                updateApplicationSensorData("package_specification", "2");
+              }}
+            />
+          </RadioGroup>
+        </Grid>
+
+        <Field
+          fieldName="Package names"
+          recoilState={applicationSensorState}
+          field="package_names"
+          inputLabel="Package names from google store"
+        />
+
+        <Grid>
+          <p className="explanation">
+            You may leave the field blank if default is selected. Please list
```

Comment:
Could change this to "You can leave the field blank if 'Default track all packages' is selected. Please list the package names, separated by a comma or space."